### ISSUE TITLE
Help Form

### ISSUE THREAD

**Issue by federicoast** [kleros/gtcr#84]
I'm thinking of some user that may think like this: "I would really like to have a list of X, but I have no idea of how to set the parameters, etc.".

Maybe we can create a Google form that the user can fill explaining some of the key features of his use case: "I'm building a decentralized e-commerce platform and I want a way to define the items to be accepted in the list".

After the user explains his use case, we can get back to him with some guidance on how to choose parameters.

![FireShot Capture 1042 - Kleros · Curate - https___curate kleros io_factory](https://user-images.githubusercontent.com/16290963/83654448-88ac7300-a593-11ea-8cc5-e56fc7eebb64.jpg)

**Comment by zambian** [comment]
I'd direct them to the Telegram group Clement setup called 'Integrations'. There is almost always someone online from the team who could be able to answer promptly.

**Comment by federicoast** [comment]
Yes, that's cool. 

I was thinking using a form so we get a more structured input from the user with all the information. If the user can already send some key information, we will be more effective in helping him.

Maybe we should have both options: the Telegram link and the Form.

@whwgeorge if we wanted users to fill a form to send us important information for us to help them estimate parameters, what should be the fields of this form? What information do we need to make this parameter estimation more effective from our side?

**Comment by mtsalenc** [comment]
Currently, the "Get help" Link at the top bar directs the user to the general Kleros telegram channel. Should we direct them to the integration channel instead? What is the link?
![Screenshot from 2020-06-03 17-16-45](https://user-images.githubusercontent.com/9872474/83684803-071e0a80-a5be-11ea-8d63-220ccae049bb.png)

**Comment by federicoast** [comment]
Yes, I think it's better if it takes you to the integration channel. This is the link: https://t.me/KlerosIntegration

**Comment by whwgeorge** [comment]
If someone asked us to help them estimate the parameters for their list, I would ask them the following:

1. What skills do you think are necessary to evaluate entries on your list?
2. What is an hourly wage at which you would expect to be able to hire a person having the required skills?
3. How much time do you think a person with the required skills would need to evaluate an entry on your list?
4. How clear cut are the criteria for membership on your list? Specifically, what percentage of the time do you think two qualified people who both put in the effort to evaluate an entry will disagree about whether it belongs on your list?
5. Can a qualified person quickly get a sense of whether an entry is suspicious and worthy of further scrutiny based on a preliminary analysis, or does one need to do a full, thorough analysis before having any sense of whether the entry is correct or incorrect? (For example, for a TCR of scam tokens, one might have a developed intuition allowing to pick out which entries “seem” like scams and are worthy of further scrutiny. For a TCR of ERC20 tokens, the deviations from the requirements tend to be subtle and would be hard to flag based on anything less than a full analysis.) 
6. If applicable, estimate a value (for example, in ETH, EUR, or USD) that it would be worth for someone to place a malicious element on your list.
7. What is your tolerance for mislabeled entries? For example, is a single (blatantly) incorrect registered entry on your list catastrophic, or would your list be useful as long as your list is “mostly” correct? Estimate if possible a percentage value (e.g. <5%) for the number of entries on your list that are incorrect at any given time that you would consider acceptable.
